examples/translate-all/widget.json

```diff
@@ -3,5 +3,5 @@
   "name": "Translate All Content",
   "src": "http://localhost:3000/",
   "sidebar": true,
-  "fieldTypes": ["Symbol", "Text"]
+  "fieldTypes": ["Symbol", "Text", "Object"]
```

Comment:
For sidebar widgets it doesn't actually make sense to restrict the field types at all, does it? But then, it isn't intuitive at all how we enable sidebar widgets in the CMApp to be used in a CT in the first place.